modules/backend/behaviors/FormController.php

```diff
@@ -763,8 +771,10 @@ public function formFindModelObject($recordId)
         /*
          * Prepare query and find model record
          */
+        /** @var \October\Rain\Database\Builder $query */
         $query = $model->newQuery();
         $this->controller->formExtendQuery($query);
+        /** @var \October\Rain\Database\Model $result */
```

Comment:
What is the point of these comments?

Reply:
Because it is an illuminate function which returns as type hint an illuminate model instead of the correct october model

Reply:
With this docblock the typehint is restored to the correct model

Reply:
@jacobdekeizer I get your reasoning, but why does this need to be specified when normally you would not interact with these variables at all besides through the `formExtendQuery` and `formExtendModel` methods which are already correctly type-hinted?

Reply:
@bennothommo You're right, these docblocks can be removed.